src/views/user/LoginView.vue

```diff
@@ -23,6 +23,16 @@
         icon="key"
       />
 
+      <div class="field">
+        <div class="control">
+          <label class="checkbox" v-if="showLoginTerms">
+            <input type="checkbox" v-model="loginTermsAgreed" />
+            <span class="has-text-weight-bold" v-translate>{{ termsMessage }}</span>
+            (<router-link :to="termsRoute" v-translate>link</router-link>).
```

Comment:
link is not supposed to be transated?

Reply:
What do you mean ? `link` will be translated here isn't it ?

Reply:
sorry, I assume I misread. however, the name is not explicit enough for the translation system. Either rename it or add a context, please.